Dart/testData/analysisServer/callHierarchy/MultiCallees/MultiCallees_verification.xml

```diff
@@ -0,0 +1,93 @@
+<node text="A.a() → B A" base="true">
+  <node text="A.a() → B A">
+    <node text="A.a() → B A">
+      <node text="A.a() → B A">
+        <node text="A.a() → B A">
+          <node text="A.a() → B A">
+            <node text="A.a() → B A">
+              <node text="A.a() → B A">
+                <node text="A.a() → B A">
+                  <node text="A.a() → B A">
+                    <node text="A.a() → B A">
+                      <Probably infinite part skipped>
+                      <Probably infinite part skipped>
+                      <Probably infinite part skipped>
```

Comment:
Thanks. I tried to find something that wouldn't cause warnings or errors to be printed but did not discover `<any/>`.